ENIGMAsystem/SHELL/Universal_System/Extensions/ParticleSystems/PS_particle_bridge_Direct3D11.h

```diff
@@ -33,26 +34,22 @@
 namespace enigma {
   namespace particle_bridge {
     void initialize_particle_bridge() {} // Do nothing, nothing to initialize.
-    inline int __GETR(int x) {return x & 0x0000FF;}
-    inline int __GETG(int x) {return (x & 0x00FF00) >> 8;}
-    inline int __GETB(int x) {return (x & 0xFF0000) >> 16;}
     double wiggle;
     int subimage_index;
     double x_offset;
     double y_offset;
-    
+
     static void draw_particle(particle_instance* it)
```

Comment:
I'd as soon have had you remove the newlines entirely and just used {}, but whatever.

Reply:
I'll do that in another PR and just delete these particle bridges. I will rewrite them to all use the `vertex_*` or new model functions, since there's no reason they can't now. I didn't notice the stubs or I would have done that though, this is just VS Code knocking out whitespace since I opened the file.